Add RFS - Real Flight Simulator (#163)
riverar/wsa-app-compatibility: README.md

| Relay | 10.0.378 | 11 | ✅
| Remini - AI Photo Enhancer || 11 | ⚠️ | Oops! Something went wrong Your image didn't save. Please try again.
| Remote Desktop (Microsoft) | 10.0.12.1148 | 11 | ✅

> | RFS - Real Flight Simulator | 1.6.1 | 12.1 | ⚠️ | Does not work with keyboard | Works only by connecting a controller or in PCs with touch-screen functionalities

Reviewer: ```suggestion
| RFS - Real Flight Simulator | 1.6.1 | 12.1 | ⚠️ | Does not work with keyboard | Works only by connecting a controller or on PCs with touch
```
